source/Engine.cpp

```diff
@@ -139,6 +140,11 @@ namespace {
 				|| ship->Cloaking() >= 1. || ship->GetPersonality().IsMute())
 			return false;
 		
+		// Ships that don't share a language with the player shouldn't send hails.
+		const Government *gov = ship->GetGovernment();
```

Comment:
Is it possible that a ship has no government (i.e. you getting a null-pointer here)?

Reply:
A nullptr check is done further up in this function. (Moving this line up above that one as I hadn't noticed it earlier.)